client/src/Routes/Routes.tsx

```diff
@@ -31,6 +31,14 @@ const Routes: React.FC = () => {
                             </PublicRoute>
                         }
                     />
+                    <Route
+                        path={FORGOT_PATH}
+                        element={
+                            <PublicRoute>
+                                <ForgotPassword name="u" />
```

Comment:
what is name="u"

Reply:
I created a random prop type(to be changed) for this component